### ISSUE TITLE
Sort comments by net +1 + -1 count on the issue view

### ISSUE THREAD

**Issue by cirosantilli** [isaacs/github#601]
Sample URL https://github.com/isaacs/github/issues/9

Analogous to https://github.com/isaacs/github/issues/600 but for comments.

**Comment by TPS** [comment]
#527?

**Comment by cirosantilli** [comment]
@TPS true, it is close (had forgtten that lol) I think both should be kept as this mentions the difference between upvotes and downvotes and deals more precisely with the new situation. I'll link to the other on description.

**Comment by cirosantilli** [comment]
@TPS note that this is about comments, not issues. Issues here https://github.com/isaacs/github/issues/605

**Comment by TPS** [comment]
@cirosantilli I'm obviously starting to lose track of all these fine variations of these interrelated issues.

Thanks! :stuck_out_tongue_winking_eye:

**Comment by AlecBenzer** [comment]
I don't know how much sense this would make. Issue comments tend to be much more conversational and chronological (i.e., people responding back and forth to each other), unlike stackoverflow answers. I think something like the ability to pin comments (#1008) would be a reasonable compromise between the two models.

**Comment by Mottie** [comment]
I created a userscript that'll allow you to [sort comments by reaction](https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-sort-reactions). Clicking on the sort a third time will restore the initial comment order.

![](https://user-images.githubusercontent.com/136959/37052861-373a814e-2140-11e8-92f9-5c6fb57de7ba.gif)

**Comment by phil294** [comment]
This would be very useful, specially given that GitHub issue comment threads have now surpassed StackOverflow in terms of usefulness. More often than not, the answer to a library-specific problem is to be found somewhere deeply nested in some issue page. Endlessly scrolling is annoying however, a simple `Sort by 👍` would save *so* much time for everyone.

Mottie's userscript is great! But unfortunately it still requires you to repeatedly click the `Load more...` link several times, as GitHub – for some reason – even hides top voted comments by default in long threads.

Oh how I wish we had arrived in the age of decentralization already. Then all of this begging-some-platform-developers-for-new-features baloney wouldn't be necessary in the first place...
